docs/community/modular-fellows.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_label: Modular Fellows
+sidebar_label: Overview
```

Comment:
We already have an Overview page on Community channel

Reply:
I'd rename it to "Modular Fellows Overview" or "Introduction to Modular Fellows"

Reply:
I added a category for fellows and changed the original `Modular Fellows` to `Overview`, so the slug will still be `/modular-fellows`

<img width="592" alt="Screenshot 2022-12-05 at 6 11 02 PM" src="https://user-images.githubusercontent.com/46639943/205763209-72acaceb-e2ae-4181-8a61-2be1669970b2.png">